packages/bubble-dev/start-preset/src/plugins/android-web-screenshots.ts

```diff
@@ -14,6 +14,7 @@ export default (fontsDir?: string) => plugin<StartFilesProps, void>('x-ray', ({
     dpr: 3,
     extensions: [
       '.web.js',
+      '.web.jsx',
```

Comment:
why android and ios are looking for `.web`?

Reply:
`android-web`  :)